### DISCUSSION TITLE
[V5] Tests don't use a secondary database

### DISCUSSION THREAD

**Discussion by FeMaffezzolli** [adonisjs/core#1923] [Help] [answered]
Hello there,

I followed this [step by step](https://preview.adonisjs.com/blog/running-tests-in-adonisjs-v5#app) tests setup and everything seems to work fine, but I realized the tests are run using the development database, not a different one.

Is this supposed to work like this or have I missed something?

**Comment by FeMaffezzolli** [comment, marked as answer]
Well, one could simply set a different database name in the test script as below:

`"test": "PG_DB_NAME=database_tests node -r @adonisjs/assembler/build/register japaFile.ts"
`

It works perfectly fine. But I'm still unsure if this is a good approach or not.

**Reply by thetutlage** [reply]
Yes that is now you have to do it. Another way is to create `.env.testing` file the project root to override env variables for the testing environment

**Reply by FeMaffezzolli** [reply]
Alright, thanks!
